### ISSUE TITLE
Can you mimic the way the replace dialog works in windows file explorer.

### ISSUE THREAD

**Issue by angaratosurion** [files-community/Files#10595]
### What feature or improvement do you think would benefit Files?

Can you mimic the way the replace dialog works in windows file explorer.
I mean when it finds a folder that has the same name with the one you copy it asks if you want to replace it or not . 
And if it finds a files that already exists in the subfolder in the target path to re ask if you want to replace it 

### Requirements

- This proposal will accomplish X
- This proposal will accomplish Y
- This proposal will accomplish Z

### Files Version

Έκδοση: 2.3.63.0 OS Version: 10.0.22000.1281

### Windows Version

Windows 11 Pro 21H2

**Comment by yaira2** [comment]
This is already supported, can you add more context?

**Comment by angaratosurion** [comment]
When windows explorer copies or moves a folder and finds that it one with
the same name exists asks if you want to replace it.
When goes in a subfolder that it has files that already exist in the target
it asks again for the files if it want to be replaced.
Your doesn't do that. If you select replace it replaces the folder and the
files in the folder and in it's subfolder.

I suggest to have different settings for files and folders.


Στις Δευ, 28 Νοε 2022, 05:59 ο χρήστης Yair ***@***.***>
έγραψε:

**Comment by yaira2** [comment]
This is essentially the same thing as merging two folders

**Comment by angaratosurion** [comment]
Yeah but it should ask if the user wants to replace the already existed files or not

**Comment by SgtSweetieBelle** [comment]
Hi, also hoping for a Merge feature.

**Comment by SilverGreen93** [comment]
Hi, I also miss this option from Windows Explorer. If I paste a folder with the same name as the existing one, I am asked to skip, rename or replce the folder. If I choose replace, all the files inside that have the same name will be overritten, which is not what I want. I want to be asked again for the files if I want to overwrite or not those files, while just merging the outside folder.
I will go back to using Windows Explorer until this is implemented.

**Comment by ELowry** [comment]
> If I choose replace, all the files inside that have the same name will be overritten, which is not what I want. I want to be asked again for the files if I want to overwrite or not those files, while just merging the outside folder.

This is correct. There are actually two UX intents happening under a single UI dropdown here:
- The first intent is whether to _skip_ or _merge_ existing **folders** with the same names:
  - **Skip**: the incoming folder gets ignored entirely (other operations can proceed).
  - **Merge**: the contents of the incoming folder get copied into the receiving folder.
- When merging, the contents of the folder then follow the standard prompt rules, including the option to replace, ignore, or rename.

I get confused by the current implementation EVERY time that folders with the same names are involved. The question is whether to split the two steps, or try to include enough options in the dropdown to handle all cases (which must also include a toggle to decide whether to merge for just one folder conflict, or all folder conflicts, or maybe even just sub-folder conflicts within the currently prompted folder?).
